proto/server.proto

```diff
@@ -277,4 +279,17 @@ message StoreCredentialRequest {
 message StoreCredentialResponse {
   StatusCode status = 1;
   string statusMessage = 2;
+}
+
+message UnDeployStoreAppRequest {
+  string clusterID = 1;
+  string appName = 2;
+  string version = 3;
+  string namespace = 4;
+  string releaseName = 5;
```

Comment:
clusterid and releaseName is sufficient, other attributes are not needed

Reply:
app name and version is required to pull app config from store

Reply:
undeploy not required to read anything store app.
so release name and clusterid is sufficient